### ISSUE TITLE
VoiceOver does not recognize indeterminate progress status

### ISSUE THREAD

**Issue by jelbourn** [angular/material2#416]
When using VoiceOver on both progress-bar and process-circle, the screen-reader does not pick up the "indeterminate" state when mode is set to indeterminate. Instead, it simply reads "0 percent, progress indicator". 

It *does* work for `md-spinner`, which sets the mode in the constructor, where it reads "indeterminate, progress indicator".

**Comment by jelbourn** [comment]
This should have been fixed by #640
